man/rules/meson.build

```diff
@@ -718,7 +719,11 @@ manpages = [
  ['sd_machine_get_class', '3', ['sd_machine_get_ifindices'], ''],
  ['sd_notify',
   '3',
-  ['sd_notifyf', 'sd_pid_notify', 'sd_pid_notify_with_fds', 'sd_pid_notifyf', 'sd_notify_barrier'],
+  ['sd_notify_barrier',
+   'sd_notifyf',
+   'sd_pid_notify',
+   'sd_pid_notify_with_fds',
+   'sd_pid_notifyf'],
```

Comment:
Not your fault obviously, but doesn't belong here. Maybe split this out into a separate commit?

Reply:
I split this out into #15743, because it has nothing to do with the rest of this PR.